src/main/java/org/cactoos/list/LengthOf.java

```diff
@@ -44,9 +45,17 @@
 
     /**
      * Ctor.
+     * @param items The array
+     */
+    public LengthOf(final Iterable<?> items) {
+        this(items.iterator());
```

Comment:
@yegor256 the whole conception of secondary ctors is wrong then as them are useless.

Anyway, here we have 2 options: either apply PR as is (and you don't want it) or cancel it at all as it's impossible to rename both `LengthOfIterable` and `LengthOfIterator` to class with the same name `LengthOf`. WDYT?